webpack.config.js

```diff
@@ -61,8 +61,23 @@ module.exports = {
     child_process: 'empty',
     net: 'empty'
   },
-  externals: {
-    react: 'react',
-    "react-dom": 'react-dom'
-  }
+  externals: [

```

Comment:
how are we making sure other public package is built?

Reply:
we will publish it from master branch for the time being